examples/gocean/eg7/README.md

```diff
@@ -0,0 +1,92 @@
+# PSyclone GOcean Example 7
+
+**Author:** J. Henrichs, Bureau of Meteorology
+
+## Introduction
+
+This is a simple test that shows how to use the read-only verification
+support in PSyclone. It is a stand-alone program that can be compiled
+and run. 
+
+## Compilation
+A makefile is provided to compile this example. If required
+it will compile the dl_esm_inf libraryW and the read-only verification
+wrapper library. By default, the compilation uses the version
+of the dl_esm_inf library provided as a git submodule (under
+``../../../external/dl_esm_inf/finite_difference``- see
+https://psyclone-dev.readthedocs.io/en/latest/working_practises.html)
+within the PSyclone repository. You can set the environment variable
+``INF_DIR`` for the ``make`` command to pick a different version.
+
+The makefile here will invoke psyclone with the ``--profile invokes``
```

Comment:
Cut-n-paste error? It uses the `read_only_transform.py` transformation script.

Reply:
Indeed.